subsys/zigbee/cli/zigbee_cli_zcl_types.h

```diff
@@ -23,40 +24,53 @@ enum attr_req_type {
 };
 
 /* Structure used to store Read/Write Attributes request data in the context manager entry. */
-struct read_write_attr_req_data {
+struct read_write_attr_req {
 	zb_zcl_frame_direction_t direction;
 	zb_uint8_t attr_type;
 	zb_uint16_t attr_id;
 	zb_uint8_t attr_value[32];
```

Comment:
I know it's inherited after previous versions, but what about putting attr_value at the very end of this structure? In case of  the array's overflow, the req_type field will not be corrupted.

Reply:
Not sure if should change this - array's overflow should not happen in the first place 😃 
What we're discussing now is the magnitude of damage.